components/card/card.css

```diff
@@ -0,0 +1,129 @@
+
+:root{
+   --orange:orange;
+   --gold:gold;
+   --orange-dark:rgb(255, 123, 0);
+   --red:red;
+   --white:white;
+   --pink:pink;
+   --grey:grey;
+
+}
+
+.card {
+    position:relative;
+    margin-top: 20px;
```

Comment:
instead of px rem would be better

Reply:
i have made the changes,please approove my PR